sql/upgrade/1.3.3_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -112,4 +112,42 @@ CALL uc_dolphin_T_t_ds_resources_un();
 DROP PROCEDURE IF EXISTS uc_dolphin_T_t_ds_resources_un;
 
 
+drop PROCEDURE if EXISTS uc_dolphin_T_t_ds_process_definition_A_is_parallel;
+delimiter d//
+CREATE PROCEDURE uc_dolphin_T_t_ds_process_definition_A_is_parallel()
+   BEGIN
+       IF NOT EXISTS (SELECT 1 FROM information_schema.COLUMNS
+           WHERE TABLE_NAME='t_ds_process_definition'
+           AND TABLE_SCHEMA=(SELECT DATABASE())
+           AND COLUMN_NAME ='is_parallel')
+   THEN
+         alter table t_ds_process_definition add is_parallel tinyint default '1' comment '0:serial,1:parallel';
```

Comment:
why we should use the procedure to add one column?

Reply:
I copy old ds project sql script and modify. I look old script is use  procedure to add one column

Reply:
please refer to the lastest SQL script. Because the ddl SQL will be run on different DBMS, such as MySQL,Postgresql, and so on, however, they might no support procedure.

Reply:
@gabrywu    MySQL  script and Postgresql script is separate in bofore project design。
![image](https://user-images.githubusercontent.com/62982788/93727507-0bfade80-fbee-11ea-8d81-d1636afffd7f.png)

Reply:
> @gabrywu MySQL script and Postgresql script is separate in bofore project design。
> ![image](https://user-images.githubusercontent.com/62982788/93727507-0bfade80-fbee-11ea-8d81-d1636afffd7f.png)

you can  see  bofore ‘s  sql  script， I keep it only。  hi，你可以查看之前和现有的项目，在mysql 中，都是以存储过程来加字段的，在Postgresql 中是以函数的方式来加字段的。我仅仅是尊照项目的要求来的。而且  mysql和 Postgresql  加字段的脚本 本来就没法去共用，我看ds 项目中一直是分开维护的。